### PR TITLE
App bar styling

### PR DESCRIPTION
**Pull request by jcytron**
things I already know need to be changed:
- the react native paper import
- the console.log("yeet button") (have no clue why this is even in there
- the displayName for Icon Color was incorrectly changed

also this version doesn't have the android shadow bug fix yet

### CODE REVIEW

**Comment by jcytron** [comment]
[App Bar Styling](https://trello.com/c/ZU01EzzB/2280-app-bar-styling)

File: src/TextButton/TextButton.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ export default class WrappedTextButton extends Component {
   }
 
   render() {
+    console.log('yeet button')
```
Comment (scott-massey) [inline src/TextButton/TextButton.js:125]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -100,6 +100,7 @@
     "@protonapp/react-native-material-ui": "^2.0.6",
     "color": "^3.1.0",
     "react-indiana-drag-scroll": "^1.7.0",
+    "react-native-paper": "^4.7.1",
```
Comment (scott-massey) [inline package.json:103]:
```suggestion
```

**Review by scott-massey — COMMENTED** [review]
need to pull latest, run `yarn` and recommit but looks good

**Review by scott-massey — APPROVED** [review]

**Pull request merged**